website/src/pages/index.tsx

```diff
@@ -82,18 +88,19 @@ function DownloadGenericLinks() {
     </div>
   );
 }
-
 function Hero() {
   // const { siteConfig } = useDocusaurusContext();
   return (
     <section className="text-gray-600 dark:text-gray-400 body-font">
-      <div className="container mx-auto flex px-5 py-24 items-center justify-center flex-col">
+      <div className="container mx-auto flex px-5 py-24 items-center justify-center flex-col md:flex-col lg:flex-row">
         <div className="text-center lg:w-2/3 w-full">
           <h1 className="title-font sm:text-4xl text-3xl lg:text-6xl mb-4 font-medium text-gray-900 dark:text-white">
-            Containers and Kubernetes for application developers
+              Podman <br/>
+              <span>&#128156;</span> <br/> 
+              Containers
```

Comment:
It's the main message appearing on the search engine. IMHO it can't be that first